assets/_sass/components/_pagination.scss

```diff
@@ -0,0 +1,36 @@
+.ais-root__collapsed .ais-body {
+  display: none;
+}
+
+.ais-root {
+  .icon-angle-sm-right::before {
+    transition: all 0.3s ease-in-out;
+  }
+
+  &:not(.ais-root__collapsed) .icon-angle-sm-right::before {
+    transform: rotate(90deg);
+  }
+}
+
+.ais-pagination--item {
+  display: inline-block;
+  margin: 0 1rem 0 0;
+
+  &__disabled {
+    display: none;
+  }
+}
+
+.selected {
```

Comment:
I know you haven't implemented the styles yet (and so this will probably change), but heads up that this is too generic of a class selector. You'll want to make it more specific to avoid any conflicts with the rest of the code base.